ansible_base/authentication/backend.py

```diff
@@ -55,6 +55,9 @@ def authenticate(self, request, *args, **kwargs):
                     )
                     return None
 
-                logger.info(f'User {user.username} logged in from authenticator with ID "{authenticator_id}"')
+                if hasattr(user, "last_login_from"):
+                    logger.info(f'User {user.username} logged in from authenticator with ID "{authenticator_id}"')
```

Comment:
I'm not sure we want the info msg to appear only if hasattr is true, maybe move it out of the if block?  (though I'm not sure when/why last_login_from wouldn't be a user attribute)